multiversx/mx-chain-go
# Update enable epochs flags

## Reasoning behind the pull request
- Added missing enable epoch flags on api endpoint

## Testing procedure
- Check that `/network/enable-epochs` returns all enable epoch falgs from `enableEpochs.toml` config file (https://github.com/multiversx/mx-chain-go/blob/rc/v1.7.next1/cmd/node/config/enableEpochs.toml)
    - for both observer nodes and proxy
- it should work with latest proxy version

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
